deploy/pre-deployment/pre-deployment-check.sh

```
@@ -175,6 +204,113 @@ check_cluster_resources() {
+        return 1
+    fi
+
+    # Count running pods
+    local running_pods
+    running_pods=$(echo "$intel_driver_pods" | awk '$4 == "Running"' | wc -l | tr -d ' ')
+
+    local total_pods
+    total_pods=$(echo "$intel_driver_pods" | wc -l | tr -d ' ')
+
+    if [[ $running_pods -eq 0 ]]; then
+        print_status $RED "❌ Intel GPU resource driver pods are not running"
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)